kettle/make_db.py

```diff
@@ -12,19 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Generates a JSON file containing test history for the last day.
+"""Generates a SQLite DB containing test data downloaded from GCS."""
 
-Writes the JSON out to tests.json.
-"""
+# pylint: disable=invalid-name,global-statement
```

Comment:
Please do not disable these for the whole file (and can we just fix them instead?)